libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -252,14 +252,35 @@ export interface BucketDeleteOptions {
   readonly mustExist?: boolean;
 }
 
+/**
+ * Specifies the HTTP method permitted by a presigned URL for a bucket.
+ */
+export enum BucketSignedUrlAction {
+  /**
+   * Represents a GET request for a presigned URL, allowing read access for an object in the bucket.
+   */
+  GET = "GET",
+  /**
+   * Represents a PUT request for a presigned URL, allowing write access for an object in the bucket.
+   */
+  PUT = "PUT",
+}
```

Comment:
For naming, I'm wondering if we should use a higher level mental model that doesn't refer to low-level HTTP verbs, e.g.
```
enum BucketSignedUrlAccess {
  READ,
  WRITE,
}
```

For example, in my head my coworker doesn't give me "put" access to their google docs spreadsheet, they give me "write" access, so it may be more common / easier to understand for anyone new to signed urls. We could still mention the HTTP verbs in the docstrings.

What do you think @garysassano @tsuf239? (I'm also totally cool with keeping the current naming)

Reply:
@Chriscbr  Actually, [I originally named those DOWNLOAD and UPLOAD](https://github.com/winglang/wing/pull/4802/commits/e008284160bc0c5121af29c393a980372e4a632d#diff-27f918bd13c7dacebbb394122669f25c09bf03e2b1f3eff4da3a4c487c0db0e1R267-R270), since I thought those are two actions any user would be familiar with, even without any knowledge about the underlying HTTP methods used for the operations.

Reply:
I'm going to restore the old naming, as I think that effectively conveys the purpose of the generated presigned URL, which is to allow you to either download or upload a file to a private bucket.